Makefile

```diff
@@ -8,11 +8,23 @@ tests: lint ## run the tests
 	python3.7 -m pytest -v jupyterfs/tests --cov=jupyterfs --junitxml=python_junit.xml --cov-report=xml --cov-branch
 	yarn test
 
-lint: ## run linter
+lintjs: ## run linter
+	yarn lint
+
+lintpy: ## run linter
 	flake8 jupyterfs setup.py
+
+lints: ## run linter
+	flake8 jupyterfs
 	yarn lint
 
-fix:  ## run autopep8/tslint fix
+fixjs:  ## run autopep8/tslint fix
+	./node_modules/.bin/tslint --fix src/*
+
+fixpy:  ## run autopep8/tslint fix
+	autopep8 --in-place -r -a -a jupyterfs/
+
+fixes:  ## run autopep8/tslint fix
```

Comment:
change to `fix` for consistency (e.g. with `nbcelltests`)

Reply:
I'm happy to change this. The reason why I named the targets `lints` and `fixes` was for consistency with `tests`. Should we also change `tests` => `test` (here and in `nbcelltests` too)?

Reply:
that sounds fine

Reply:
done